admin/assets/stylesheets/elements/_gds-app-cookie-banner.scss

```diff
@@ -0,0 +1,53 @@
+.app-cookie-banner {
+  font-family: "nta",Arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 14px;
+  font-size: .875rem;
+  line-height: 1.14286;
+  color: #0b0c0c;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  width: 100%;
+  padding-top: 15px;
```

Comment:
Have you checked this layout on mobile and tablet views?

Reply:
@jon-shipley checked using dev tools on different sizes and it looks good even on small mobile devices. Will keep a note for further testing using browserstack as well.